### PR TITLE
Helm + deploy for admin map

### PR DESCRIPTION
**Pull request by james-otten**
Add helm chart and deploy to k8s clusters

- [ ] Update `ADMIN_FQDN` and `FQDN` in both environments

### CODE REVIEW

File: infra/admin-map/values.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
```
Comment (WillNilges) [inline infra/admin-map/values.yaml:33]: delete

File: .github/workflows/helm-deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+name: Deploy Helm Chart
```
Comment (WillNilges) [inline .github/workflows/helm-deploy.yaml:1]: This _could_ be a reusable workflow

File: .github/workflows/helm-deploy.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          helm upgrade \
+            adminmap \
+            ./infra/admin-map \
+            --install \
+            --kubeconfig ./config \
+            --kube-apiserver https://${{ secrets.SSH_TARGET_IP }}:6443 \
+            -n ${{ vars.APP_NAMESPACE }} \
+            --create-namespace \
+            -f ./infra/admin-map/values.yaml \
+            --set bookstack.fqdn="${{ vars.FQDN }}"
```
Comment (WillNilges) [inline .github/workflows/helm-deploy.yaml:49]: bookstack?

**Review by WillNilges — APPROVED** [review]
A couple things but I understand if we wanna just get the fckin thing working

File: .github/workflows/publish-and-deploy.yaml [hunk 4]
```diff
@@ -5,11 +5,13 @@ on:
     branches:
       #- meshdb
       - meshdb-admin
+      - james/admin-helm
```
Comment (WillNilges) [inline .github/workflows/publish-and-deploy.yaml:8]: delete this

File: .github/workflows/helm-deploy.yaml (hunk 3 above)
Comment (james-otten) [reply .github/workflows/helm-deploy.yaml]: Guess where I copy/pasted from. Good catch

File: infra/admin-map/values.yaml (hunk 1 above)
Comment (james-otten) [reply infra/admin-map/values.yaml]: done

**Pull request merged**